synchronicity/combined_types.py

```diff
@@ -0,0 +1,69 @@
+import functools
+import typing_extensions
+from synchronicity.async_wrap import wraps_by_interface
+from synchronicity.exceptions import UserCodeException
+from synchronicity.interface import Interface
+import typing
+
+if typing.TYPE_CHECKING:
+    from synchronicity.synchronizer import Synchronizer
+
+
+class FunctionWithAio:
```

Comment:
FunctionWithAio and MethodWithAio were just moved here from the synchronizer, they are unchanged FYI